test/cli/test-fixtures/image-java-spdx-tools/Makefile

```diff
@@ -0,0 +1,8 @@
+all: build validate
+
+.PHONY: build
+build:
+	docker build -t spdx-java-tools:latest .
+
+validate:
+	docker run --rm -v ${FILE}:/home/build/${BASE} spdx-java-tools:latest Verify ${BASE}
```

Comment:
Is this just a utility to run the validator manually?

Reply:
the test calls it here:
https://github.com/anchore/syft/blob/026428f53d62350d1053f36af03c05655e1b346e/test/cli/spdx_tooling_validation_test.go#L74-L81

Reply:
The weirdness with the renaming is because the spdx validation tools only take a file with the `.spdx` suffix so the temp files as they are made will fail if consumed by the validator